openadapt/strategies/mixins/mpt7b.py

```diff
@@ -0,0 +1,38 @@
+"""
+Implements a ReplayStrategy mixin which gets completions from a 4-bit quantized
+version of MPT-7b, for inference on CPU.
+
+
+Usage:
+
+    class MyReplayStrategy(MPT7BReplayStrategy):
+        ...
+"""
+
+from ctransformers import AutoModelForCausalLM
+from openadapt.models import Recording
+from openadapt.strategies.base import BaseReplayStrategy
+
+
+class MPT7BReplayStrategy(BaseReplayStrategy):
+    """
+    MPT-7B Replay Strategy for inference on CPU.
+    """
+
+    def __init__(
+        self,
+        recording: Recording
+    ):
+        super.__init__(recording)
+        self.model_path = 'TheBloke/MPT-7B-GGML'
+        self.model_file = 'mpt-7b.ggmlv3.q4_0.bin'
```

Comment:
Please define constants at the top of the file, and pass in as default arguments here, e.g.:

```
    def __init__(
        self,
        recording: Recording,
        model_path: str = MODEL_PATH,
        model_file: str = MODEL_FILE,
    ):
    ...
    self.model_path = model_path
    self.model_file = model_file
```